temporalcli/commands.operator_namespace.go

```diff
@@ -0,0 +1,335 @@
+			Namespace:        nsName,
+			PromoteNamespace: true,
+		}
+	} else if len(c.ActiveCluster) > 0 {
+		fmt.Printf("Will set active cluster name to: %s, other flag will be omitted.\n", c.ActiveCluster)
+		replicationConfig := &replication.NamespaceReplicationConfig{
+			ActiveClusterName: c.ActiveCluster,
+		}
+		updateRequest = &workflowservice.UpdateNamespaceRequest{
+			Namespace:         nsName,
+			ReplicationConfig: replicationConfig,
+		}
+	} else {
+		resp, err := cl.WorkflowService().DescribeNamespace(cctx, &workflowservice.DescribeNamespaceRequest{
+			Namespace: nsName,
+		})
+		if err != nil {
+			switch err.(type) {
+			case *serviceerror.NamespaceNotFound:
+				return err
+			default:
+				return fmt.Errorf("namespace update failed: %w", err)
+			}
+		}
+
+		description := resp.NamespaceInfo.GetDescription()
+		ownerEmail := resp.NamespaceInfo.GetOwnerEmail()
+		retention := resp.Config.GetWorkflowExecutionRetentionTtl()
+
+		if len(c.Description) > 0 {
+			description = c.Description
+		}
+		if len(c.Email) > 0 {
+			ownerEmail = c.Email
+		}
+
+		data := map[string]string{}
+		if len(c.Data) > 0 {
+			data, err = stringKeysValues(c.Data)
+			if err != nil {
+				return err
+			}
+		}
+
+		if len(c.Retention) > 0 {
+			ret, err := timestamp.ParseDurationDefaultDays(c.Retention)
+			if err != nil {
+				return fmt.Errorf("option retention format is invalid: %w", err)
+			}
+			retention = durationpb.New(ret)
+		}
+
+		var clusters []*replication.ClusterReplicationConfig
+		if len(c.Cluster) > 0 {
+			for _, clusterName := range c.Cluster {
+				clusters = append(clusters, &replication.ClusterReplicationConfig{
+					ClusterName: clusterName,
+				})
+			}
+		}
+
+		updateInfo := &namespace.UpdateNamespaceInfo{
+			Description: description,
+			OwnerEmail:  ownerEmail,
+			Data:        data,
+		}
+
+		updateConfig := &namespace.NamespaceConfig{
+			WorkflowExecutionRetentionTtl: retention,
+			HistoryArchivalState:          archivalState(c.HistoryArchivalState.String()),
+			HistoryArchivalUri:            c.HistoryUri,
+			VisibilityArchivalState:       archivalState(c.VisibilityArchivalState.String()),
+			VisibilityArchivalUri:         c.VisibilityUri,
+		}
+		replicationConfig := &replication.NamespaceReplicationConfig{
+			Clusters: clusters,
+		}
+		updateRequest = &workflowservice.UpdateNamespaceRequest{
+			Namespace:         nsName,
+			UpdateInfo:        updateInfo,
+			Config:            updateConfig,
+			ReplicationConfig: replicationConfig,
+		}
+	}
+
+	if c.Verbose {
+		_ = cctx.Printer.PrintStructured(updateRequest, printer.StructuredOptions{})
+	}
+
+	_, err = cl.WorkflowService().UpdateNamespace(cctx, updateRequest)
+	if err != nil {
+		switch err.(type) {
+		case *serviceerror.NamespaceNotFound:
+			return err
+		default:
+			return fmt.Errorf("namespace update failed: %w", err)
+		}
+	} else {
+		cctx.Printer.Println(color.GreenString("namespace %s update succeeded.", nsName))
+	}
```

Comment:
```suggestion
	}
	cctx.Printer.Println(color.GreenString("Namespace %s update succeeded.", nsName))
```

No need for `else`, and capitalizing success message as you have in other commands